### PR TITLE
Video listing page

### PR DESCRIPTION
**Pull request by sharathnair9999**
Completed Landing Page and Video Listing page

- Added Few more videos to DB
- Video Card with Skeleton Loading
- Empty Data Component in case there are no videos
- Select Category of the videos and filter accordingly
- Sort by latest and oldest videos

Pending:

- Add Card actions, Date of Upload and Video Duration to the Video Card

### CODE REVIEW

File: src/components/CategoryChip/CategoryChip.js [hunk 1]
```diff
@@ -10,13 +10,14 @@ const CategoryChip = ({ category }) => {
   } = videosState;
 
   const selectCategory = (cat) => {
+    console.log("category", cat);
```
Comment (AmishMishra11) [inline src/components/CategoryChip/CategoryChip.js]: remove console

**Review by AmishMishra11 — APPROVED** [review]
Nice clean code
good folder structure
liked how you have exported components

File: src/components/CategoryChip/CategoryChip.js (hunk 1 above)
Comment (sharathnair9999) [reply src/components/CategoryChip/CategoryChip.js]: Oops!!  yup will do that

File: src/App.js [hunk 2]
```diff
@@ -25,14 +25,18 @@ const LazyLanding = React.lazy(() => import("./pages/Landing/Landing"));
 
 function App() {
   const location = useLocation();
+  const mainContainerRef = useRef();
```
Comment (ArjunGTX) [inline src/App.js:28]: good that you have used `ref` instead of directly accessing the dom element

File: src/pages/VideoListing/VideoListing.css [hunk 3]
```diff
@@ -23,13 +26,33 @@
...
+  place-items: center;
+  padding: 0 0 10px 0;
+}
 
-  .categories-chip-container{
-    gap: 8px;
-  }
+.sort-items{
+  gap: 10px;
+  padding: 10px 16px;
```
Comment (ArjunGTX) [inline src/pages/VideoListing/VideoListing.css:39]: better to use rem unit for values greater than 10px

**Review by ArjunGTX — APPROVED** [review]
Code looks good. One thing I noticed is that, the folder names and file names are in `camelCase`. it is better to use `lower-case` for folder names and file names other than react component files

**Review by SukanyaSen530 — APPROVED** [review]

**Pull request merged**